### PR TITLE
Remove scheduler throttling.

### PR DESCRIPTION
**Pull request by jasonwyatt**
This should help resolve issues with performance variance.

### CODE REVIEW

File: javatests/arcs/core/allocator/AllocatorTestBase.kt [hunk 1]
```diff
@@ -132,6 +135,12 @@ open class AllocatorTestBase {
         WritePerson.throws = false
     }
 
+    @After
+    fun tearDown() {
+        schedulerProvider.cancelAll()
+        Thread.sleep(1000)
```
Comment (ianull) [inline javatests/arcs/core/allocator/AllocatorTestBase.kt:141]: When and why do we need a sleep like this?

If it's for waiting all jobs done on its scheduler, we should wait and complete all jobs while responding to the cancelAll() w/ or w/out a timeout.

Comment (jasonwyatt) [reply javatests/arcs/core/allocator/AllocatorTestBase.kt:141]: I agree completely. I've been fighting with flakiness in this test class while trying to get this PR to pass. I'll just start @Ignore'ing the flakes.  I've got a task when I get time to work on it to audit all of our coroutine launches and determine how to make everything cancelable and scoped correctly.

**Comment by ianull** [comment]
Perfect!

--Ian

File: javatests/arcs/core/allocator/AllocatorTestBase.kt (hunk 1 above)
Comment (ianull) [reply javatests/arcs/core/allocator/AllocatorTestBase.kt:141]: Perfect!

**Pull request closed without merging**